### PR TITLE
D8CORE-1382: News packaging. (MOVED TO #17)

### PR DESCRIPTION
**Pull request by sherakama**
# (MOVED TO #17)

### CODE REVIEW

**Comment by boznik** [comment]
I'm getting an error on install:
[Drupal\Core\Entity\EntityStorageException (23000)]                                                                                                
  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '11' for key 'PRIMARY': INSERT INTO {file_managed} (fid, uuid, langcode, ui  
  d, filename, uri, filemime, filesize, status, created, changed) VALUES (:db_insert_placeholder_0, :db_insert_placeholder_1, :db_insert_placeholde  
  r_2, :db_insert_placeholder_3, :db_insert_placeholder_4, :db_insert_placeholder_5, :db_insert_placeholder_6, :db_insert_placeholder_7, :db_insert  
  _placeholder_8, :db_insert_placeholder_9, :db_insert_placeholder_10); Array                                                                        
  (                                                                                                                                                  
      [:db_insert_placeholder_0] => 11                                                                                                               
      [:db_insert_placeholder_1] => d2860dd3-4e1c-4d05-bc16-c3df65594cec                                                                             
      [:db_insert_placeholder_2] => en                                                                                                               
      [:db_insert_placeholder_3] => 1                                                                                                                
      [:db_insert_placeholder_4] => banner-151017-3191.jpg                                                                                           
      [:db_insert_placeholder_5] => public://media/image/banner-151017-3191.jpg                                                                      
      [:db_insert_placeholder_6] => image/jpeg                                                                                                       
      [:db_insert_placeholder_7] => 266909                                                                                                           
      [:db_insert_placeholder_8] => 1                                                                                                                
      [:db_insert_placeholder_9] => 1581635659                                                                                                       
      [:db_insert_placeholder_10] => 1581635659

**Comment by sherakama** [comment]
> @sherakama
> I'm getting an error on install:
> [Drupal\Core\Entity\EntityStorageException (23000)]
> SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '11' for key 'PRIMARY': INSERT INTO {file_managed} 

Make sure you have the right branch and patch for the default content found in the stanford_news module here: https://github.com/SU-SOE/stanford_news/blob/default-content/composer.json#L37

**Comment by boznik** [comment]
ok, that did it. i had to run a composer update...

File: config/sync/core.entity_view_display.paragraph.stanford_media_caption.default.yml [hunk 1]
```diff
@@ -61,9 +61,9 @@ content:
     label: hidden
     settings:
       image:
-        image_formatter: image_style
```
Comment (sherakama) [inline config/sync/core.entity_view_display.paragraph.stanford_media_caption.default.yml:64]: Snuck this in here as it really should have been a responsive image to begin with.

File: config/sync/page_manager.page.node_view.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+uuid: 93fde1a7-b8c7-449c-920e-d41a1f6a907a
+langcode: en
+status: false
```
Comment (sherakama) [inline config/sync/page_manager.page.node_view.yml:3]: Basically, all I am doing is turning this off.

File: config/sync/user.role.site_builder.yml [hunk 3]
```diff
@@ -34,7 +34,6 @@ permissions:
   - 'administer crop types'
   - 'administer display modes'
   - 'administer entity browsers'
-  - 'administer footer menu items'
```
Comment (sherakama) [inline config/sync/user.role.site_builder.yml:37]: We are not using this footer menu. We have a global footer and a local footer that uses other configurations.

**Comment by sherakama** [comment]
@pookmish 

This is the sister PR to https://github.com/SU-SWS/stanford_profile/pull/194

**Pull request closed without merging**